osu.Game/Skinning/LegacySkinConfiguration.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+namespace osu.Game.Skinning
+{
+    public class LegacySkinConfiguration : DefaultSkinConfiguration
+    {
+        public const decimal LATEST_VERSION = 2.5m;
```

Comment:
should be 2.7

![image](https://user-images.githubusercontent.com/191335/66531899-623f5080-eb48-11e9-8646-de2524c0a68f.png)

Reply:
Oops. I've read on the wiki that the latest version is `2.5`, should probably be updated.